### ISSUE TITLE
warn if DOI already exists in database

### ISSUE THREAD

**Issue by beuerle** [pubs/pubs#236]
Currently pubs adds a new publication with a given DOI even if it already exists in the repository, generating a new entry and bibkey.

It would be good if pubs could give a warning, or ask if the publication should be added even if it is already present?

**Comment by benureau** [comment]
Yes, I agree. To be fast, we would probably need to have a cache of the DOIs present in the repository. Will take some work.

**Comment by boutil** [comment]
This would be great to avoid adding duplicate.
The same kind of check would be nice and useful with arXiv ids for example.
